src/core/app/Server.ts

```diff
@@ -47,7 +47,7 @@ export class Server {
 
   private static matchServerURL(buffer: string | Buffer) {
     const str = buffer.toString().trim();
-    const m = str.match(/Storybook started on => (https?:\/\/.+)/);
+    const m = str.match(/(?:started(?:.|\n)+Local:\s+)(https?:\/\/\S+)/);
```

Comment:
If we wanted something with more specificity, you could try:
```
/\s+Storybook\s\S*\sstarted\s+\│\s+\│\s+\│\s+\│\s+Local:\s+(https?:\/\/.+)/gm)
```
Which would match the leading `Storybook` message and the following empty line